### DISCUSSION TITLE
Getting the Lisp object for a HTML ID

### DISCUSSION THREAD

**Discussion by ramenbytes** [rabbibotton/clog#60] [Q&A] [answered]
Is there a way to get the Lisp object associated with a HTML ID? I know that client-side there is the `clog` array, is there an equivilant for the Lisp side? From what I've seen so far it looks like any corresponding Lisp objects are thrown away if they are not explicitly stored in a variable somewhere.

**Comment by rabbibotton** [comment, marked as answer]
The state of each html element is stored on the browser, so better to look at the lisp objects as "handles" to the html elements. So you are correct.

**Reply by ramenbytes** [reply]
Thanks. Would you be open to a pull request that creates an equivalent to the `clog` array for the Lisp side, which holds all the "handles"? My intent is to use it to implement an analogue to "copy object to repl" like Sly/Slime has, except for GUIs. So for example, I right click something in the browser window and select "Copy Object to REPL" and the Lisp "handle" pops up in my Sly REPL in emacs.

**Reply by rabbibotton** [reply]
Sounds useful, for sure interested.
